transports/sftp/src/main/java/org/mule/transport/sftp/SftpMessageReceiver.java

```diff
@@ -332,4 +350,24 @@ protected void doDispose()
     {
         // no op
     }
+
+    private class SFTPMessageFactory extends DefaultMuleMessageFactory
```

Comment:
Yes. I'm sure. There are overriden methods in DefaultMuleMessageFactory that are necessary.